spec/helpers/checkouts_helper_spec.rb

```diff
@@ -78,7 +78,8 @@
 
     it 'renders the right html' do
       checkbox = '<input type="checkbox" name="renewal_list[]" id="renewal_list_" '
-      checkbox += "value=\"#{checkout.item_key}\" class=\"checkbox\" multiple=\"multiple\" />"
+      checkbox += "value=\"#{checkout.item_key}\" data-checkbox-type=\"renewal\" "
+      checkbox += 'class="checkbox" multiple="multiple" />'
```

Comment:
Do you like this better? I think the switching between single and double quotes is a little weird? Maybe? 🤷‍♂ 

```suggestion
            checkbox = "<input type=\"checkbox\" name=\"renewal_list[]\" id=\"renewal_list_\" "\
                 "value=\"#{checkout.item_key}\" data-checkbox-type=\"renewal\" class=\"checkbox\" "\
                 "multiple=\"multiple\" />"
```